content/posts/android/Installing Kali Nethunter Lite on DivestOS 20.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Installing Kali Nethunter Lite on DivestOS 20"
+date: 2022-07-18
+tags: ['Operating Systems', 'Android', 'Security']
+author: Tommy
+---
+
+**Disclaimer**: This will turn your device into a purely penetration testing device, and it will be very insecure for any other purposes. Please do not do this on your daily driver.
+
+![Kali Nethunter on a OnePlus 7T](/images/kali-nethunter-op7t.jpg)
+
+Originally, I just needed a device to run packet capture on my other devices. I have a OnePlus 7T laying around which I am not using for anything (since it has already reached its end-of-life), so I thought it would be cool to give it a new purpose. I am using DivestOS here since it is my go-to distribution, but you can use LineageOS or any other distribution if you want to.
```

Comment:
```suggestion
Originally, I just needed a device to run packet capture on my other devices. I have a OnePlus 7T laying around which I am not using for anything (since it has already reached its end-of-life), so I thought it would be cool to give it a new purpose. I am using DivestOS here since it is my go&#8209;to Android distribution, but you can use LineageOS or any other distribution if you want to.
```